lib/src/base.dart

```diff
@@ -891,10 +893,12 @@ class Runtime {
   final String rawDescription;
 
   const Runtime({this.key, this.name, this.version, this.rawDescription})
+      // ignore: prefer_is_empty
```

Comment:
If the `prefer_is_empty` suggestion is taken here it fails with `Invalid constant value.dart(invalid_constant)`.

Reply:
Actually this one is affecting the score on pub.dev:

<img width="424" alt="image" src="https://user-images.githubusercontent.com/1633368/79083609-bd274f80-7d05-11ea-9fcb-152c18cbbbe7.png">

Would be nice to find a solution

Reply:
Sounds like a gap in the language. /cc @lrhn @leafpetersen @eernstg

Reply:
Sorry, `.length` is literally called out specially in the spec as one of the only things you're allowed to do in constants.  Expanding the scope of what you can do in constants is a frequent request, so noted.

Reply:
Ah, then it's a linter issue. The linter should not be in a conflict with the language. Which quickly led me to https://github.com/dart-lang/linter/issues/1719